src/actions.cpp

```diff
@@ -452,6 +457,19 @@ void evm_contract::process_tx(const runtime_config& rc, eosio::name miner, const
 
     engine.finalize(ep.state(), ep.evm().block());
     ep.state().write_to_db(ep.evm().block().header.number);
+
+    if (gas_param_pair.second) {
+        const auto& gas_param_v1 = std::get<gas_parameter_type::gas_parameter_data_v1>(gas_param_pair.first);
+        action(std::vector<permission_level>{}, get_self(), "configchgdv1"_n, 
```

Comment:
It would be better to use an `action_wrapper` to have type safety.

Also, the `gas_price` needs to be included as well in this event.